Photon-Tinker/AppDelegate.swift

```diff
@@ -31,6 +31,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         LogList.startLogging()
         LogList.clearStaleLogs()
 
+        NSLog("ParticleCloud.sharedInstance().accessToken = \(ParticleCloud.sharedInstance().accessToken as Optional)")
```

Comment:
This doesn't use `ParticleLogger`.  Will that log to the logs we share to support?

Reply:
Yep, it will show in the console but not in the logs.